tests/test_port_dpb_neigh.py

```diff
@@ -0,0 +1,241 @@
+from swsscommon import swsscommon
+import redis
+import time
+import os
+import pytest
+import json
+import re
+from port_dpb import DPB
+
+@pytest.mark.usefixtures('dpb_setup_fixture')
+class TestPortDPBNeigh(object):
+    def clear_srv_config(self, dvs):
+        dvs.servers[0].runcmd("ip address flush dev eth0")
+        dvs.servers[1].runcmd("ip address flush dev eth0")
+
+    def test_set_neighbor_DPB(self, dvs, testlog):
```

Comment:
Write descriptions for each below test cases what your intention and expected results.